### PR TITLE
ELEMENTS-1454: pdf.js update to v2.12.313

### PR DESCRIPTION
**Pull request by richardsd**
Following the work done for ELEMENTS-1415 and ELEMENTS-1454 in 0.1 nuxeo-ui-elements branch, this PR updates the PDF.js version used in 3.0.x to the latest version (v2.12.313) without legacy ES5 support.

With this PR, we ensure that we are using the same PDF.js version across the different versions of elements/web ui.

### CODE REVIEW

**Review by mnixo — DISMISSED** [review]

**Review by mnixo — APPROVED** [review]

File: polymer.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "lint": {
     "rules": ["polymer-3"],
-    "filesToIgnore": ["**/node_modules/**/*.js"],
+    "filesToIgnore": ["**/node_modules/**/*.js", "**/ui/viewers/pdfjs/**/**/*.js"],
```
Comment (richardsd) [inline polymer.json:4]: @Gabez0r tried different paths here, but if it starts with `ui/...` the files are not ignored 🤔

Comment (Gabez0r) [reply polymer.json:4]: Does it work with `"**/ui/viewers/pdfjs/**/**/*.js"`?

Comment (Gabez0r) [reply polymer.json:4]: If it does, I'm fine with it!

**Review by Gabez0r — APPROVED** [review]

File: polymer.json (hunk 1 above)
Comment (richardsd) [reply polymer.json:4]: Yes, everything is working with `**/ui/viewers/pdfjs/**/**/*.js`.

**Comment by richardsd** [comment]
Cross repo build https://github.com/nuxeo/nuxeo-web-ui/actions/runs/1659377751 🔵

**Pull request closed without merging**